### ISSUE TITLE
rspec focus check not picking up syntactic sugar versions of focus

### ISSUE THREAD

**Issue by prpetten** [jish/pre-commit#205]
I typically just add 'f' to the beginning of my specs to add focus when I run guard.  So 'describe' becomes 'fdescribe', 'context' becomes 'fcontext', and 'it' becomes 'fit'.  This achieves the same result as adding :focus, but I don't want it in any of my commits either (nor does anyone else on my team).

**Comment by prpetten** [comment]
^ The above pull request should fix this issue once it is merged.

**Comment by mpapis** [comment]
closing in favor of #204 as it shows some code that can be reused (especially tests)
